### PR TITLE
Give the `README.md` a lick of paint 🎨 

### LINKED ISSUE #38: Give the `README.md` a lick of paint 🎨
#### Brief 💯 

A good `README.md` goes a long way for new contributors and downstream consumers of the plugin. I would recommend we align the repository `README.md` to the format used in https://github.com/citi/citi-ospo.

#### Tasklist ✅ 

- [x] Use the [Citi/citi-ospo](https://github.com/citi/citi-ospo) `README.md` format
- [x] Move the `CONTRIBUTING.md` to the root of the repository
- [x] Move the `PUBLISHING.md` to the root of the repository
- [x] Include a reference to Code of Conduct
- [x] Include link to Gradle Plugins website

### PR DESCRIPTION
**Pull request by JamieSlome**
Closes #38

### CODE REVIEW

File: helm-plugin/src/test/kotlin/com/citi/gradle/plugins/helm/PluginVersionSynchronization.kt [hunk 1]
```diff
@@ -15,7 +15,7 @@ class PluginVersionSynchronization {
         val pluginVersion = getPluginVersion()
         val expectedDocumentationLine = ":version: $pluginVersion"
 
-        val readmeFile = File("../README.adoc").absoluteFile.readText()
+        val readmeFile = File("../README.md").absoluteFile.readText()
```
Comment (JamieSlome) [inline helm-plugin/src/test/kotlin/com/citi/gradle/plugins/helm/PluginVersionSynchronization.kt:18]: @imanushin - would this be causing a breakage in the CI?

**Comment by magner669** [comment]
Please provide test evidence that we can build docs.

**Comment by JamieSlome** [comment]
@magner669 - updated the unit test 👍 Ready to merge if you are happy ❤️

File: PUBLISHING.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This page describes how to publish a new version of plugin and all related artif
 
 Steps:
 
-1. Create a PR with version increment in [gradle.properties](../gradle.properties) and in [documentation](../README.adoc). Please follow [semver guidelines](https://semver.org/):
+1. Create a PR with version increment in [gradle.properties](../gradle.properties) and in [documentation](../README.md). Please follow [semver guidelines](https://semver.org/):
```
Comment (magner669) [inline PUBLISHING.md:7]: Links to files need fixing

File: README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+- Fix typos or touch up documentation
+- Share your opinions on [existing issues](https://github.com/citi/gradle-helm-plugin/issues)
+- Help expand and improve our library by [opening a new issue](https://github.com/citi/gradle-helm-plugin/issues/new)
+
+Please review our [community contribution guidelines](https://github.com/Citi/.github/blob/main/CONTRIBUTING.md) and [functional contribution guidelines](./CONTRIBUTING.md) to get started 👍
+
+## Code of Conduct
+
+We are committed to making open source an enjoyable and respectful experience for our community. See <a href="https://github.com/Citi/.github/blob/main/CODE_OF_CONDUCT.md"><code>CODE_OF_CONDUCT</code></a> for more information.
```
Comment (magner669) [inline README.md:84]: Why is HTML A used here instead of markdown link `[foo](bar.com)`

Comment (magner669) [reply README.md:84]: and in line 88

**Review by magner669 — CHANGES_REQUESTED** [review]

File: PUBLISHING.md (hunk 2 above)
Comment (JamieSlome) [reply PUBLISHING.md:7]: It has already been fixed; I renamed the file `README.adoc` to `README.md` and this has been shown as a change 👍

File: README.md (hunk 3 above)
Comment (JamieSlome) [reply README.md]: Sorted on both line 84 & 88 ❤️

File: PUBLISHING.md (hunk 2 above)
Comment (magner669) [reply PUBLISHING.md:7]: Yes , but you also moved the file to the parent directory.  You need to remove ../

**Comment by JamieSlome** [comment]
@magner669 - updated the file references as mentioned - apologies for the misunderstanding on my end! ❤️

**Review by magner669 — APPROVED** [review]

**Pull request merged**